src/Microsoft.ML.TimeSeries/SequentialAnomalyDetectionTransformBase.cs

```diff
@@ -306,10 +307,10 @@ public abstract class AnomalyDetectionStateBase : StateBase
             protected SequentialAnomalyDetectionTransformBase<TInput, TState> Parent;
 
             // A windowed buffer to cache the update values to the martingale score in the log scale.
-            private FixedSizeQueue<Double> _logMartingaleUpdateBuffer;
+            private FixedSizeQueue<Double> LogMartingaleUpdateBuffer { get; set; }
```

Comment:
>LogMartingaleUpdateBuffer [](start = 43, length = 25)

why this change?

Reply:
also, could you add this assembly to be tracked by RoboTom?

---
In reply to: [236008010](https://github.com/dotnet/machinelearning/pull/1618#discussion_r236008010) [](ancestors = 236008010)